packages/contractkit/package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Now would be a good opportunity to specify a homepage property, maybe keywords too. For example:
https://github.com/babel/babel/blob/master/packages/babel-core/package.json#L7
https://www.npmjs.com/package/@babel/core

Reply:
done